### DISCUSSION TITLE
Allow setting first day of the week in calendar interface

### DISCUSSION THREAD

**Discussion by wc-davide** [directus/directus#12874] [Feature Requests]
Add a new setting that allows users to change the first day of the week for the calendar interface:
![image](https://user-images.githubusercontent.com/46521712/164199513-bdbdf0c7-3d21-47c1-93b6-2d9b85aa4d57.png)
(screenshot is in italian, but it's just to explain which interface I'm referring to)

Currently the weeks are displayed Sunday to Saturday, but here in italy we use Monday to Sunday so an option to allow setting Monday as the first day of the week would be really useful.

Another way to implement this could be to just follow the user locale/language, example my locale is italian so it would make sense to display the calendar following the usual italian format (monday to sunday)

**Comment by azrikahar** [comment]
Would be nice indeed. Sort of a similar thing was asked for the Calendar layout here: https://github.com/directus/directus/discussions/5842#discussioncomment-2393869, and it was implemented in #12613 to be configurable by the user.

However, with https://github.com/directus/directus/discussions/5842#discussioncomment-2408344 in mind for this, where some user may want `en-US` locale but Monday as first day, I'm wondering how can we make this configurable for such scenario (at least other than straight up adding an extra field in user profile page).

**Reply by benhaynes** [reply]
Adding this as a setting on the User Detail is the best option I can think of... and perhaps having a global setting for a fallback.

**Reply by rijkvanzanten** [reply]
Is there ever a case where you'd want different datetime fields to start the week on different days? I doubt it, but just thinking aloud about possible edge cases

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
